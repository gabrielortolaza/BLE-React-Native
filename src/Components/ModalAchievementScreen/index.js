/** Page appears to no longer be used */
/* eslint-disable */
// import React, { Component } from 'react'
// import {
//   View,
//   Image,
//   Animated,
//   Dimensions,
//   Text,
//   Modal
// } from 'react-native'
// import PropTypes from 'prop-types'
// import LinearGradient from 'react-native-linear-gradient'
// // import LottieView from 'lottie-react-native'

// import StyleSheet from '../../Proportional'
// import { RoundButton } from '../Shared/index'
// import Fonts from '../../Themes/Fonts'

// const Lighting1 = require('../../Assets/Animations/Lighting1.json')
// const Lighting2 = require('../../Assets/Animations/Lighting2.json')

// export const getAchievementImage = achievementId => {
//   switch (achievementId) {
//     case 'age_1':
//     case 'streak_1':
//     case 'volume_1':
//     case 'badge_1':
//       return require('../../Assets/Images/Milestones/step1.png')
//     case 'age_2':
//     case 'streak_2':
//     case 'volume_2':
//     case 'volume_3':
//     case 'badge_2':
//       return require('../../Assets/Images/Milestones/step2.png')
//     case 'age_3':
//     case 'badge_3':
//       return require('../../Assets/Images/Milestones/step3.png')
//     case 'age_4':
//     case 'badge_4':
//       return require('../../Assets/Images/Milestones/step4.png')
//     case 'age_5':
//     case 'badge_5':
//       return require('../../Assets/Images/Milestones/step5.png')
//   }
// }

// export default class ModalAchievementScreen extends Component {
//   state = {
//     animation: new Animated.Value(0)
//   }

//   static navigationOptions = {
//     header: null
//   }

//   setAnim = (anim) => {
//     if (!this.anim) {
//       this.anim = anim
//       this.anim.play()
//     }
//   }

//   onPressOk = () => {
//     if (this.props.achievementDisplayed) {
//       this.props.achievementDisplayed(this.props.achievement.key)
//     }
//   }

//   render () {
//     if (!this.props.achievement) {
//       return null
//     }
//     const {
//       key,
//       texts
//     } = this.props.achievement

//     if (!key || !texts) {
//       return null
//     }

//     const {
//       allCenter,
//       title1,
//       title2,
//       subtitle,
//       description
//     } = texts

//     const altLayout = texts.altLayout && !allCenter

//     return (
//       <Modal
//         onDismiss={this.onPressOk}
//         onRequestClose={this.onPressOk}
//         testID='AchievementScreen'
//       >
//         <LinearGradient
//           colors={['rgb(133, 155, 224)', 'rgb(82, 176, 208)']}
//           style={styles.container}
//           testID='AchievementScreen_Gradient'
//         >
//           {
//             // <LottieView
//             //   ref={this.setAnim}
//             //   source={altLayout ? Lighting2 : Lighting1}
//             //   resizeMode='cover'
//             //   contentMode='aspectFill'
//             //   style={StyleSheet.absoluteFill}
//             //   testID={altLayout ? 'AchievementScreen_LottieAlt' : 'AchievementScreen_Lottie'}
//             //   autoSize
//             // />
//           }
//           {!altLayout && (
//             <View
//               style={styles.top}
//               testID='AchievementScreen_Top'
//             >
//               <Text
//                 style={[styles.title, allCenter && styles.titleCenter]}
//                 testID={allCenter ? 'AchievementScreen_TopTitleCenter' : 'AchievementScreen_TopTitle'}
//               >{title1}</Text>
//               {(!!subtitle && !allCenter) && (
//                 <Text
//                   style={styles.subtitle}
//                   testID='AchievementScreen_TopSubtitle'
//                 >{subtitle}</Text>
//               )}
//             </View>
//           )}
//           <View
//             style={altLayout ? styles.imageRight : styles.imageCenter}
//             testID={altLayout ? 'AchievementScreen_AltImage' : 'AchievementScreen_CenterImage'}
//           >
//             {altLayout && (
//               <Text
//                 style={[styles.title, styles.titleWithImage]}
//                 testID='AchievementScreen_AltTitle'
//               >{title1}</Text>
//             )}
//             <Image
//               testID='AchievementScreen_Image'
//               source={getAchievementImage(key)}
//             />
//           </View>
//           {altLayout && (
//             <View
//               style={styles.bottom}
//               testID='AchievementScreen_Bottom'
//             >
//               <Text
//                 style={styles.title}
//                 testID='AchievementScreen_BottomTitle'
//               >{title2}</Text>
//               <Text
//                 style={styles.subtitle}
//                 testID='AchievementScreen_BottomSubtitle'
//               >{subtitle}</Text>
//             </View>
//           )}
//           <View
//             style={[styles.footer, allCenter && styles.footerCenter]}
//             testID={allCenter ? 'AchievementScreen_FooterCenter' : 'AchievementScreen_Footer'}
//           >
//             <Text
//               style={[styles.description, !allCenter && styles.descriptionFullWidth]}
//               testID={allCenter ? 'AchievementScreen_DescriptionCenter' : 'AchievementScreen_Description'}
//             >{description}</Text>
//             <RoundButton
//               minimal
//               onPress={this.onPressOk}
//               small
//               testID='AchievementScreen_Ok'
//               white
//             >Okay!</RoundButton>
//           </View>
//         </LinearGradient>
//       </Modal>
//     )
//   }
// }

// const styles = StyleSheet.createProportional({
//   container: {
//     flex: 1,
//     backgroundColor: 'transparent',
//     paddingVertical: 35
//   },
//   title: {
//     ...Fonts.Light,
//     fontSize: 50,
//     lineHeight: 60,
//     color: 'white',
//     paddingTop: 10
//   },
//   titleCenter: {
//     paddingTop: 10,
//     textAlign: 'center'
//   },
//   titleWithImage: {
//     position: 'absolute',
//     left: 40,
//     bottom: -5
//   },
//   subtitle: {
//     ...Fonts.SemiBold,
//     fontSize: 14,
//     color: 'white',
//     lineHeight: 20
//   },
//   top: {
//     paddingHorizontal: 40
//   },
//   bottom: {
//     paddingBottom: 20,
//     paddingHorizontal: 40
//   },
//   imageCenter: {
//     flex: 1,
//     justifyContent: 'center',
//     alignItems: 'center'
//   },
//   imageRight: {
//     flex: 1,
//     flexDirection: 'row',
//     justifyContent: 'flex-end',
//     alignItems: 'flex-end',
//     paddingBottom: 10
//   },
//   description: {
//     ...Fonts.Light,
//     fontSize: 12,
//     color: 'white',
//     marginRight: 18,
//     minHeight: 30
//   },
//   descriptionFullWidth: {
//     flex: 1,
//     minHeight: 38
//   },
//   footer: {
//     flexDirection: 'row',
//     alignItems: 'flex-end',
//     paddingHorizontal: 40
//   },
//   footerCenter: {
//     paddingTop: 15,
//     flexDirection: 'column',
//     alignItems: 'center',
//     justifyContent: 'center'
//   },
//   backgroundImage: {
//     width: Dimensions.get('screen').width,
//     height: Dimensions.get('screen').height
//   }
// })

// ModalAchievementScreen.propTypes = {
//   achievementDisplayed: PropTypes.func,
//   achievement: PropTypes.shape({
//     key: PropTypes.string,
//     texts: PropTypes.shape({
//       altLayout: PropTypes.bool,
//       allCenter: PropTypes.bool,
//       title1: PropTypes.string,
//       title2: PropTypes.string,
//       subtitle: PropTypes.string,
//       description: PropTypes.string
//     })
//   })
// }
