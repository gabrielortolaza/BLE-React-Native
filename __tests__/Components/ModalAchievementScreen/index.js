import React from 'react'
import { shallow } from 'enzyme'

import ModalAchievementScreen, { getAchievementImage } from '../../../src/Components/ModalAchievementScreen'

describe('ModalAchievementScreen', () => {
  it('Empty 1', () => {
    const wrapper = shallow(
      <ModalAchievementScreen />)

    expect(wrapper.isEmptyRender()).toBeTruthy()
    expect(wrapper).toMatchSnapshot()
  })

  it('Empty 2', () => {
    const wrapper = shallow(
      <ModalAchievementScreen achievement={{ key: 'someKey' }} />)

    expect(wrapper.isEmptyRender()).toBeTruthy()
    expect(wrapper).toMatchSnapshot()
  })

  it('Empty 3', () => {
    const wrapper = shallow(
      <ModalAchievementScreen achievement={{ texts: {} }} />)

    expect(wrapper.isEmptyRender()).toBeTruthy()
    expect(wrapper).toMatchSnapshot()
  })

  it('Structure - Button', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()

    expect(wrapper.isEmptyRender()).toBeFalsy()

    const okButton = render.find('RoundButton')
    expect(okButton).toHaveLength(1)

    expect(achievementDisplayed.mock.calls).toHaveLength(0)
    okButton.simulate('press')
    expect(achievementDisplayed.mock.calls).toHaveLength(1)

    expect(render).toMatchSnapshot()
  })

  it('Structure - Default', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()

    // Base
    expect(wrapper.isEmptyRender()).toBeFalsy()
    expect(render.find({ testID: 'AchievementScreen' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Gradient' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Image' })).toHaveLength(1)
    expect(render.find('RoundButton')).toHaveLength(1)

    // Default Layout
    expect(render.find({ testID: 'AchievementScreen_Lottie' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Top' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_TopTitle' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_TopSubtitle' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_CenterImage' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Footer' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Description' })).toHaveLength(1)

    // Alt Layout
    expect(render.find({ testID: 'AchievementScreen_LottieAlt' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_AltTitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_AltImage' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_Bottom' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_BottomTitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_BottomSubtitle' })).toHaveLength(0)

    // Center Layout
    expect(render.find({ testID: 'AchievementScreen_TopTitleCenter' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_FooterCenter' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_DescriptionCenter' })).toHaveLength(0)

    expect(render).toMatchSnapshot()
  })

  it('Structure - altLayout', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`,
        altLayout: true
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()

    // Base
    expect(wrapper.isEmptyRender()).toBeFalsy()
    expect(render.find({ testID: 'AchievementScreen' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Gradient' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Image' })).toHaveLength(1)
    expect(render.find('RoundButton')).toHaveLength(1)

    // Default Layout
    expect(render.find({ testID: 'AchievementScreen_Lottie' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_Top' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_TopTitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_TopSubtitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_CenterImage' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_Footer' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Description' })).toHaveLength(1)

    // Alt Layout
    expect(render.find({ testID: 'AchievementScreen_LottieAlt' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_AltTitle' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_AltImage' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Bottom' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_BottomTitle' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_BottomSubtitle' })).toHaveLength(1)

    // Center Layout
    expect(render.find({ testID: 'AchievementScreen_TopTitleCenter' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_FooterCenter' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_DescriptionCenter' })).toHaveLength(0)

    expect(render).toMatchSnapshot()
  })

  it('Structure - allCenter', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`,
        allCenter: true
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()

    // Base
    expect(wrapper.isEmptyRender()).toBeFalsy()
    expect(render.find({ testID: 'AchievementScreen' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Gradient' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Image' })).toHaveLength(1)
    expect(render.find('RoundButton')).toHaveLength(1)

    // Default Layout
    expect(render.find({ testID: 'AchievementScreen_Lottie' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Top' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_TopTitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_TopSubtitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_CenterImage' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_Footer' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_Description' })).toHaveLength(0)

    // Alt Layout
    expect(render.find({ testID: 'AchievementScreen_LottieAlt' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_AltTitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_AltImage' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_Bottom' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_BottomTitle' })).toHaveLength(0)
    expect(render.find({ testID: 'AchievementScreen_BottomSubtitle' })).toHaveLength(0)

    // Center Layout
    expect(render.find({ testID: 'AchievementScreen_TopTitleCenter' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_FooterCenter' })).toHaveLength(1)
    expect(render.find({ testID: 'AchievementScreen_DescriptionCenter' })).toHaveLength(1)

    expect(render).toMatchSnapshot()
  })

  it('Contents - Default', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />
    )

    expect(wrapper.find({ testID: 'AchievementScreen_TopTitle' }).render().text()).toBe(achievement.texts.title1)
    expect(wrapper.find({ testID: 'AchievementScreen_TopSubtitle' }).render().text()).toBe(achievement.texts.subtitle)
    expect(wrapper.find({ testID: 'AchievementScreen_Description' }).render().text()).toBe(achievement.texts.description)
  })

  it('Contents - altLayout', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        title2: `Title2`,
        subtitle: `Subtitle`,
        description: `Description`,
        altLayout: true
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)

    // Alt Layout
    expect(wrapper.find({ testID: 'AchievementScreen_AltTitle' }).render().text()).toBe(achievement.texts.title1)
    expect(wrapper.find({ testID: 'AchievementScreen_BottomTitle' }).render().text()).toBe(achievement.texts.title2)
    expect(wrapper.find({ testID: 'AchievementScreen_BottomSubtitle' }).render().text()).toBe(achievement.texts.subtitle)
    expect(wrapper.find({ testID: 'AchievementScreen_Description' }).render().text()).toBe(achievement.texts.description)
  })

  it('Contents - allCenter', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`,
        allCenter: true
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)

    expect(wrapper.find({ testID: 'AchievementScreen_TopTitleCenter' }).render().text()).toBe(achievement.texts.title1)
    expect(wrapper.find({ testID: 'AchievementScreen_DescriptionCenter' }).render().text()).toBe(achievement.texts.description)
  })

  it('Badge 1', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_1'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()
    const image = render.find({ testID: 'AchievementScreen_Image' })
    expect(image).toHaveLength(1)
    const sourceProp = image.prop('source')

    const badge = getAchievementImage('badge_1')
    const badgeAge = getAchievementImage('age_1')
    const badgeStreak = getAchievementImage('streak_1')
    const badgeVolume = getAchievementImage('volume_1')

    expect(sourceProp.testUri).toBe(badge.testUri)
    expect(badgeAge.testUri).toBe(badge.testUri)
    expect(badgeStreak.testUri).toBe(badge.testUri)
    expect(badgeVolume.testUri).toBe(badge.testUri)
  })

  it('Badge 2', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_2'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()
    const image = render.find({ testID: 'AchievementScreen_Image' })
    expect(image).toHaveLength(1)
    const sourceProp = image.prop('source')

    const badge = getAchievementImage('badge_2')
    const badgeAge = getAchievementImage('age_2')
    const badgeStreak = getAchievementImage('streak_2')
    const badgeVolume2 = getAchievementImage('volume_2')
    const badgeVolume3 = getAchievementImage('volume_3')

    expect(sourceProp.testUri).toBe(badge.testUri)
    expect(badgeAge.testUri).toBe(badge.testUri)
    expect(badgeStreak.testUri).toBe(badge.testUri)
    expect(badgeVolume2.testUri).toBe(badge.testUri)
    expect(badgeVolume3.testUri).toBe(badge.testUri)
  })

  it('Badge 3', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_3'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()
    const image = render.find({ testID: 'AchievementScreen_Image' })
    expect(image).toHaveLength(1)
    const sourceProp = image.prop('source')

    const badge = getAchievementImage('badge_3')
    const badgeAge = getAchievementImage('age_3')

    expect(sourceProp.testUri).toBe(badge.testUri)
    expect(badgeAge.testUri).toBe(badge.testUri)
  })

  it('Badge 4', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_4'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()
    const image = render.find({ testID: 'AchievementScreen_Image' })
    expect(image).toHaveLength(1)
    const sourceProp = image.prop('source')

    const badge = getAchievementImage('badge_4')
    const badgeAge = getAchievementImage('age_4')

    expect(sourceProp.testUri).toBe(badge.testUri)
    expect(badgeAge.testUri).toBe(badge.testUri)
  })

  it('Badge 5', () => {
    const achievementDisplayed = jest.fn()
    const key = 'badge_5'
    const achievement = {
      key,
      texts: {
        title1: `Title1`,
        subtitle: `Subtitle`,
        description: `Description`
      }
    }
    const wrapper = shallow(
      <ModalAchievementScreen
        achievement={achievement}
        achievementDisplayed={achievementDisplayed}
      />)
    const render = wrapper.dive()
    const image = render.find({ testID: 'AchievementScreen_Image' })
    expect(image).toHaveLength(1)
    const sourceProp = image.prop('source')

    const badge = getAchievementImage('badge_5')
    const badgeAge = getAchievementImage('age_5')

    expect(sourceProp.testUri).toBe(badge.testUri)
    expect(badgeAge.testUri).toBe(badge.testUri)
  })
})
