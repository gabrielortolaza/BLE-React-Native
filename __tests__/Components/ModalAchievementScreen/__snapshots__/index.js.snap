// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalAchievementScreen Empty 1 1`] = `ShallowWrapper {}`;

exports[`ModalAchievementScreen Empty 2 1`] = `ShallowWrapper {}`;

exports[`ModalAchievementScreen Empty 3 1`] = `ShallowWrapper {}`;

exports[`ModalAchievementScreen Structure - Button 1`] = `ShallowWrapper {}`;

exports[`ModalAchievementScreen Structure - Default 1`] = `ShallowWrapper {}`;

exports[`ModalAchievementScreen Structure - allCenter 1`] = `ShallowWrapper {}`;

exports[`ModalAchievementScreen Structure - altLayout 1`] = `ShallowWrapper {}`;
